ExtLibs/MV04.Camera/CameraHandler.cs

```diff
@@ -155,6 +161,8 @@ public string MediaSavePath
         }
 
         private Timer RecordingTimer;
+        
+        public static string url = $"rtspsrc location=rtsp://{SettingManager.Get(Setting.CameraStreamIP)}:{SettingManager.Get(Setting.CameraStreamPort)}/live0 latency=0 ! application/x-rtp ! rtph265depay ! avdec_h265 ! videoconvert ! video/x-raw,format=BGRA ! appsink name=outsink";               //@"rtspsrc location=rtsp://192.168.0.203:554/live0 latency=0 ! application/x-rtp ! rtph265depay ! avdec_h265 ! videoconvert ! video/x-raw,format=BGRA ! appsink name=outsink";              //@"videotestsrc ! video/x-raw, width=1920, height=1080, framerate=30/1 ! videoconvert ! video/x-raw,format=BGRA ! appsink name=outsink";
```

Comment:
Ezt ki kéne szervezni az MV04.Settings-be a CameraStreamIP és Port helyére

Reply:
Az IP és Port változhat a többi állandó. Miért kéne kiszervezni?

Reply:
Oké ezt csak félreolvastam

Reply:
Ebben az esetben viszont a port nem változó, de a csatorna igen...